dev/MenuBar/MenuBarItem.xaml

```diff
@@ -8,6 +8,8 @@
         <Setter Property="Background" Value="{ThemeResource MenuBarItemBackground}"/>
         <Setter Property="BorderThickness" Value="{ThemeResource MenuBarItemBorderThickness}"/>
         <Setter Property="BorderBrush" Value="{ThemeResource MenuBarItemBorderBrush}"/>
+        <Setter Property="Margin" Value="{StaticResource MenuBarItemMargin}"/>
+        <Setter Property="contract7Present:CornerRadius" Value="{StaticResource MenuBarItemCornerRadius}"/>
```

Comment:
You should use `ThemeResource` binding instead because StaticResource will be evaluated only once so if you have different scenarios where these resources will be overriden by the user, it will not be updated after the first time template was applied :)